ui/app/src/model/auth-client.ts

```diff
@@ -29,9 +30,17 @@ export interface AuthBody {
   password: string;
 }
 
+export function useIsTokenExist() {
```

Comment:
I don't really know what is returning useJwt, could we use the decoded payload for the check? At least we know the payload format is fine

Reply:
well if there is no token then there is decoded payload ;)